src/posts/development_notam-2021-06-20.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Development Updates June 2021'
+date: '2021-06-20'
+authors:
+  - 'Valastiri'
+  - 'Falcon'
+  - 'Davy'
+  - 'Boris'
+  - 'Kevin'
+category: 'DEVELOPMENT UPDATE'
+metaImage: '/img/notam-images/efb-featured.png'
+metaAlt: 'FlyByWire Simulations'
+---
+
+## Overview
+
+It has been an amazing journey over these past ten months here at FlyByWire Simulations. We have some exciting things that are released on our development branch! In this NOTAM we are going to provide you with an overview of changes we have implemented towards the following:
+
+- flyPadOS v2! Brand new UI and features available.
+- Sound implementation + upcoming planned audio effects
+- Hydraulics implementation and state
+
+---
+
+## Development Updates
+
+### flyPadOS v2
+
+The latest version of our flyPad has the following main features:
+
+- Landing performance calculator
+- New user interface design
+- More options in the settings page
+- Navigraph charts integration
+
+You heard it right! We’re excited to deliver Navigraph charts functionality in the EFB. Getting started is simple - all you need to do is to:
+
+1. Select the Charts tab in the EFB
+2. Scan the QR code
+
+NOTE: The use of Navigraph charts requires an active Ultimate subscription with Navigraph. You can check the current price at [Navigraph Subscriptions](https://navigraph.com/products/subscriptions).
+
+We are also planning to add Chartfox integration to the EFB some time later.
+
+As briefly touched on before, we have done a major redesign to the look and feel of the entire EFB for a more refined appearance and improved usability.
+
+While exploring the EFB you may have noticed that the settings page is now no longer greyed out. We have moved all the settings from the MCDU into the EFB. On a final note, the default keybinds no longer trigger when using an input while utilizing the EFB natively, meaning you no longer have to “pop-out” the EFB window.
+
+
+[Credit to the developers who worked on FPOSv2] 
```

Comment:
Can we find out who was involved so they can have more credibility?

Reply:
This is pending till I get more information, since this portion was drafted by Falcon.